ombott/static_stream.py

```diff
@@ -50,6 +51,25 @@ def _file_iter_range(fp, offset, bytes_len, maxread=1024 * 1024):
         part = fp.read(min(bytes_len, maxread))
 
 
+def make_disposition_header(file_name, disposition_type="attachment"):
+    """
+    Set content headers according to RFC 5987
+
+    :param file_name: str|unicode
+    :param disposition_type: str
+    :return:
+    """
+    ascii_name = (
+        unicodedata.normalize("NFKD", file_name).encode("ascii", "ignore").decode()
```

Comment:
I think it's better  `.encode("ascii", "replace")` - i.e. replace non-ascii characters with `?` instead of removing